<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="FTR3COF93BK8A76" name="TestCases.WFA_LSMS_LNP_1" tests="1" failures="1" timestamp="2022-07-19T03:46:43 EDT" time="36.659" errors="0">
  <testcase name="LSMS_LNP_008_Pooled_account_SNET" time="36.659" classname="TestCases.WFA_LSMS_LNP_1">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	@@line_start [364] line_end; expected [This is a pooled account. Do not discuss Pooled status with the customer.] but found [This is a pooled account. Do not discussPooled status with the customer.]gui_start  gui_end,
	@@line_start [374] line_end; expected [CUSTOMER DOES NOT AGREE] but found [The Tree no display!]gui_start  gui_end">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
@@line_start [364] line_end; expected [This is a pooled account. Do not discuss Pooled status with the customer.] but found [This is a pooled account. Do not discussPooled status with the customer.]gui_start  gui_end,
@@line_start [374] line_end; expected [CUSTOMER DOES NOT AGREE] but found [The Tree no display!]gui_start  gui_end
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at TestCases.WFA_LSMS_LNP_1.LSMS_LNP_008_Pooled_account_SNET(WFA_LSMS_LNP_1.java:390)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- LSMS_LNP_008_Pooled_account_SNET -->
  <system-out/>
</testsuite> <!-- TestCases.WFA_LSMS_LNP_1 -->
